src/crawler.js

```diff
@@ -137,22 +141,33 @@ Crawler.prototype.crawl = function(ipp, hops) {
   var self = this;
   self.queued[ipp] = REQUEST_STATUS.REQUESTING;
 
-  self.crawlOne(ipp, function(err, response, body) {
+  self.crawlOne(ipp, function(error, response, body) {
     self.dequeue(ipp);
 
-    if (err) {
-      self.logger.error(ipp + ' has err ', err);
-      self.errors[ipp] = err.code;
+    if (error) {
+      // save error
+      var err = {}
+      err[ipp] = error;
+      self.errors.push(err);
```

Comment:
I think something is off here. There's a bunch of empty entries in the crawler's output's `errors` like:
`"192.170.145.70:51235": {}`
If you `console.log` the error above, you get lots of:
`[Error: 139717991397248:error:14082174:SSL routines:SSL3_CHECK_CERT_AND_ALGORITHM:dh key too small:../deps/openssl/openssl/ssl/s3_clnt.c:3377:
]`